src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { memo, useEffect } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: (errorMessage: string) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = memo(({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
```

Comment:
You should clear the timeout on the component unmount. You can read here about it https://bobbyhadz.com/blog/react-cleartimeout